mongodb/tests/main_test.bal

```diff
@@ -359,3 +306,16 @@ function testDelete() returns Error? {
 
     mongoClient->close();
 }
+
+public type Movie record {
+    string name;
+    string year;
+    int rating;
+};
+
+public type Index record {
```

Comment:
If both `Movie` and `Index` record types are used only within the test, we don't need to make them public

We can do like the following

```
type Movie record {
    string name;
    string year;
    int rating;
};
```